packages/SwingSet/src/devices/timer-src.js

```diff
@@ -275,9 +276,11 @@ export function buildRootDeviceNode(tools) {
     // the authority they represent as capabilities.
     makeRepeater(startTime, interval) {
       const index = nextRepeater;
+      Nat(startTime);
+      Nat(interval);
```

Comment:
Is there a style rule or rule of thumb that calls for pulling these out? [Not complaining, just trying to understand why this was consistently changed.]

Reply:
Nat coerces to BigInt, so if we prefer that it not be a bigInt, we can't use the resulting value. Up to you whether you would like to use Numbers or BigInts in the different timer values. Just let me know.

Reply:
I think MarkM has pointed out that small BigInts don't cost a performance penalty, so they can all be bigInts.

Reply:
Ok, sounds good